docs/version.rst

```diff
@@ -86,6 +86,18 @@ Reference
         version number, including trailing zeroes but not including the epoch
         or any prerelease/development/postrelease suffixes
 
+    .. attribute:: major
+
+        An integer giving the major component of the ``release`` tuple
```

Comment:
Since "major" and friends are a specific semver concept we should probably link to those docs as they are not discussed anywhere else in these docs and are not inherent to Python versioning.

```suggestion
        An integer giving the major component of the ``release`` tuple as defined by `semantic versioning`_.
```
With the appropriate addition of the link to https://semver.org/. Same goes for the two other attributes.

Reply:
Hi @brettcannon 

So do I add the same semver link(https://semver.org/) to all 3 lines? Also I saw that the third component of version string is called `patch` there and not `macro`. Do we want to change that?

Reply:
@brettcannon I think it'd be a good idea to avoid referencing SemVer (since PEP 440 != SemVer), for all the same reasons as listed in https://github.com/pypa/packaging/pull/229#issuecomment-582775979.

It'd be better to instead describe these attributes in one-big-swoopwith something like the following:

> While any number of additional components after the first are permitted under PEP 440, the most common variants are to use two components ("major.minor") or three components ("major.minor.micro"). The following properties provide easier access to these components.

Reply:
> Do we want to change that?

No. See #229. :)

Reply:
But where would that paragraph go? Sphinx docs don't really flow that way and I wouldn't want the `release` section of the docs to be dependent on the order the other attributes are documented.

If we are going to avoid mentioning semver because people keep asking for **all** of semver when we do, maybe the docs on each line should just say:
```suggestion
        An integer representing the first item of :attr:`release` or `0` if unavailable.
```

Reply:
I think we can update the `.. attribute:: release` section to add the paragraph suggested by @pradyunsg, maybe add the link to `semantic versioning` if we want to.

And we keep the new section descriptions to `An integer representing the first item of :attr:`release` or ``0`` if unavailable.`, as per @brettcannon 's suggestion

This way we can keep Sphinx docs happy and also include both suggestions :) What do you guys think?

Reply:
We don't both need to be happy, we just both need to agree. 😉 What are your thoughts on my proposal, @pradyunsg ?

Definitely skip the semver link regardless of whose approach we go with.

Reply:
> We don't both need to be happy, we just both need to agree.

I agree. 😛 

> the docs on each line should just say:

Works for me!

Reply:
Thanks @pradyunsg and @brettcannon ,
I have added the requested changes, please have a look.